src/main/scala/io/iohk/praos/crypto/package.scala

```diff
@@ -1,21 +1,22 @@
 package io.iohk.praos
 
 import akka.util.ByteString
-
 import scala.util.Random
 import scala.math.abs
 
+
 package object crypto {
+
   type Key = ByteString
   type KeyPair = (Key, Key)
   type RandomValue = Int
 
   // Dummy implementation to simulate a public-private key schema.
   val keyLength = 32
 
-  private def genKey = (s: RandomValue, length: Int) => ByteString(Array.fill(length)(s.toByte))
+  def genKey = (s: RandomValue, length: Int) => ByteString(Array.fill(length)(s.toByte))
```

Comment:
Minor detail: Why is this function no longer `private`? It seems to only be used in this file.

Reply:
Correct, fixed.